pymongo/mongo_client.py

```diff
@@ -1142,22 +1146,33 @@ def max_pool_size(self):
         ``waitQueueTimeoutMS`` is set, a blocked operation will raise
         :exc:`~pymongo.errors.ConnectionFailure` after a timeout.
         By default ``waitQueueTimeoutMS`` is not set.
+
+        .. versionchanged:: 3.13
+           Deprecated.  Use ``client.options.pool_options.max_pool_size`` instead.
```

Comment:
It would be nice to add the `Use ``client.options.pool_options.max_pool_size`` instead` to the warnings too.

Reply:
Updated